bcolz/tests/test_ctable.py

```diff
@@ -19,7 +19,10 @@
 from bcolz.py2help import xrange, PY2
 import pickle
 import os
-from mock import Mock
```

Comment:
I didn't notice we had something for that, I'll move those changes to a new PR, thanks!